test/models/pool_test.rb

```diff
@@ -134,6 +134,25 @@ def test_import_hosts
       assert @pool_one.subscription_facets.where(id: host.subscription_facet.id).any?
     end
 
+    def test_import_hosts_no_consumers
```

Comment:
Added a few new test cases to make sure I didn't break anything. These pass on master too